scripts/export_demucs_onnx.py

```
@@ -1,260 +1,438 @@
+assert test_out.shape[0] == 1 and test_out.shape[1] == 1, \
+    f'Unexpected output shape: {test_out.shape}'
+
+print('[4/8] torch.jit.trace...')
 try:
     traced = torch.jit.trace(wrapper, dummy, strict=False)
-    print('[4/8] jit.trace succeeded')
+    with torch.no_grad():
+        verify = traced(dummy)
+    print(f'      Traced output: {list(verify.shape)}  '
+          f'min={float(verify.min()):.4f}  max={float(verify.max()):.4f}')
 except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)